store/tikv/backoff_test.go

```diff
@@ -37,6 +37,6 @@ func (s *testBackoffSuite) TearDownTest(c *C) {
 func (s *testBackoffSuite) TestBackoffWithMax(c *C) {
 	b := NewBackoffer(context.TODO(), 2000)
 	err := b.BackoffWithMaxSleep(boTxnLockFast, 30, errors.New("test"))
-	c.Assert(err, NotNil)
+	c.Assert(err, IsNil)
```

Comment:
why this test result is changed?

Reply:
This test case fails but never runs once before :(